server/coordinator.go

```diff
@@ -130,6 +132,11 @@ func (c *coordinator) patrolRegions() {
 			continue
 		}
 
+		// Check merge operator.
+		if c.limiter.OperatorCount(schedule.OpMerge) >= c.cluster.GetMergeScheduleLimit() {
+			continue
```

Comment:
seems the above two condition will block the other unrelated checker, it's ok?

Reply:
I think not ok.